src/objecttypes/scss/admin/_admin_theme.scss

```diff
@@ -36,6 +36,12 @@ DO NOT PUT ANY TARGET APP-SPECIFIC RULES HERE.
 
   --object-tools-bg: #888;
   --object-tools-hover-bg: #{$color_dark};
+
+  // version styling
+  --version-fg: var(--body-quiet-color);
+
+  // environment styling
+  --env-button-fg: #{$color-secondary};
```

Comment:
Can these be moved to admin.css in open-api-framework or are they only available here?

Reply:
This relies on SCSS variables (`$color-secondary`) which only exist in the project itself, so I don't think we can easily move this to OAf

Reply:
Reasoning behind this was that we reuse the admin theme colors here (of the project itself) instead of possibly introducing new colors through OAF.